### ISSUE TITLE
【EESS】New submissions for Wed, 18 Oct 23

### ISSUE THREAD

**Issue by Yukeaaa** [Yukeaaa/arxiv-daily#763]
## Keyword: volume render
There is no result 
## Keyword: volumetric render
There is no result 
## Keyword: remote render
There is no result 
## Keyword: hybrid render
There is no result 
## Keyword: raycast
There is no result 
### Contrastive Self-Supervised Learning for Spatio-Temporal Analysis of  Lung Ultrasound Videos
 - **Authors:** Li Chen, Jonathan Rubin, Jiahong Ouyang, Naveen Balaraju, Shubham Patil, Courosh Mehanian, Sourabh Kulhare, Rachel Millin, Kenton W Gregory, Cynthia R Gregory, Meihua Zhu, David O Kessler, Laurie Malia, Almaz Dessie, Joni Rabiner, Di Coneybeare, Bo Shopsin, Andrew Hersh, Cristian Madar, Jeffrey Shupp, Laura S Johnson, Jacob Avila, Kristin Dwyer, Peter Weimersheimer, Balasundar Raju, Jochen Kruecker, Alvin Chen
 - **Subjects:** Image and Video Processing (eess.IV)
 - **Arxiv link:** https://arxiv.org/abs/2310.10689
 - **Pdf link:** https://arxiv.org/pdf/2310.10689
 - **Abstract**
 Self-supervised learning (SSL) methods have shown promise for medical imaging applications by learning meaningful visual representations, even when the amount of labeled data is limited. Here, we extend state-of-the-art contrastive learning SSL methods to 2D+time medical ultrasound video data by introducing a modified encoder and augmentation method capable of learning meaningful spatio-temporal representations, without requiring constraints on the input data. We evaluate our method on the challenging clinical task of identifying lung consolidations (an important pathological feature) in ultrasound videos. Using a multi-center dataset of over 27k lung ultrasound videos acquired from over 500 patients, we show that our method can significantly improve performance on downstream localization and classification of lung consolidation. Comparisons against baseline models trained without SSL show that the proposed methods are particularly advantageous when the size of labeled training data is limited (e.g., as little as 5% of the training set).
## Keyword: medical visualization
There is no result 
## Keyword: interactive volume
There is no result 
## Keyword: rendering
There is no result 
## Keyword: cinematic rendering
There is no result 
## Keyword: volume data
There is no result 
## Keyword: remote visualization
There is no result 
## Keyword: direct volume rendering
There is no result 
## Keyword: mobile device
There is no result 
## Keyword: transfer function
There is no result 
## Keyword: retrieval
There is no result 
## Keyword: video retrieval
There is no result 
### Intelligent Resource Allocation for UAV-Based Cognitive NOMA Networks:  An Active Inference Approach
 - **Authors:** Felix Obite, Ali Krayani, Atm S. Alam, Lucio Marcenaro, Arumugam Nallanathan, Carlo Regazzoni
 - **Subjects:** Signal Processing (eess.SP)
 - **Arxiv link:** https://arxiv.org/abs/2310.11070
 - **Pdf link:** https://arxiv.org/pdf/2310.11070
 - **Abstract**
 Future wireless networks will need to improve adaptive resource allocation and decision-making to handle the increasing number of intelligent devices. Unmanned aerial vehicles (UAVs) are being explored for their potential in real-time decision-making. Moreover, cognitive non-orthogonal multiple access (Cognitive-NOMA) is envisioned as a remedy to address spectrum scarcity and enable massive connectivity. This paper investigates the design of joint subchannel and power allocation in an uplink UAV-based cognitive NOMA network. We aim to maximize the cumulative sum rate by jointly optimizing the subchannel and power allocation based on the UAV's mobility at each time step. This is often formulated as an optimization problem with random variables. However, conventional optimization algorithms normally introduce significant complexity, and machine learning methods often rely on large but partially representative datasets to build solution models, assuming stationary testing data. Consequently, inference strategies for non stationary events are often overlooked. In this study, we introduce a novel active inference-based learning approach, rooted in cognitive neuroscience, to solve this complex problem. The framework involves creating a training dataset using random or iterative methods to find suboptimal resource allocations. This dataset trains a mobile UAV offline, enabling it to learn a generative model of discrete subchannels and continuous power allocation. The UAV then uses this model for online inference. The method incrementally derives new generative models from training data by identifying dynamic equilibrium conditions between required actions and variables, represented within a unique dynamic Bayesian network. The proposed approach is validated through numerical simulations, showing efficient performance compared to suboptimal baseline schemes.
## Keyword: smartphone
There is no result 
## Keyword: medical volume data
There is no result 
## Keyword: webgpu
There is no result 
## Keyword: webgl
There is no result 
## Keyword: pre-rendering
There is no result 
## Keyword: prerendering
There is no result 
## Keyword: motion prediction
There is no result 
## Keyword: incremental learning
There is no result 
## Keyword: svm incremental
There is no result 
## Keyword: nerf
There is no result 
## Keyword: multiorgan
There is no result 
## Keyword: multi-organ
There is no result 
## Keyword: multi organ
There is no result 
### Constant Modulus Waveform Design with Block-Level Interference  Exploitation for DFRC Systems
 - **Authors:** Byunghyun Lee, Anindya Bijoy Das, David J. Love, Christopher G. Brinton, James V. Krogmeier
 - **Subjects:** Signal Processing (eess.SP)
 - **Arxiv link:** https://arxiv.org/abs/2310.10804
 - **Pdf link:** https://arxiv.org/pdf/2310.10804
 - **Abstract**
 Dual-functional radar-communication (DFRC) is a promising technology where radar and communication functions operate on the same spectrum and hardware. In this paper, we propose an algorithm for designing constant modulus waveforms for DFRC systems. Particularly, we jointly optimize the correlation properties and the spatial beam pattern. For communication, we employ constructive interference-based block-level precoding (CI-BLP) to exploit distortion due to multi-user and radar transmission. We propose a majorization-minimization (MM)-based solution to the formulated problem. To accelerate convergence, we propose an improved majorizing function that leverages a novel diagonal matrix structure. We then evaluate the performance of the proposed algorithm through rigorous simulations. Simulation results demonstrate the effectiveness of the proposed approach and the proposed majorizer.
### Rapid Non-cartesian Reconstruction Using an Implicit Representation of  GROG Kernels
 - **Authors:** Daniel Abraham, Mark Nishimura, Xiaozhi Cao, Congyu Liao, Kawin Setsompop
 - **Subjects:** Signal Processing (eess.SP)
 - **Arxiv link:** https://arxiv.org/abs/2310.10823
 - **Pdf link:** https://arxiv.org/pdf/2310.10823
 - **Abstract**
 MRI data is acquired in Fourier space. Data acquisition is typically performed on a Cartesian grid in this space to enable the use of a fast Fourier transform algorithm to achieve fast and efficient reconstruction. However, it has been shown that for multiple applications, non-Cartesian data acquisition can improve the performance of MR imaging by providing fast and more efficient data acquisition, and improving motion robustness. Nonetheless, the image reconstruction process of non-Cartesian data is more involved and can be time-consuming, even through the use of efficient algorithms such as non-uniform FFT (NUFFT). This work provides an efficient approach (iGROG) to transform the non-Cartesian data into Cartesian data, to achieve simpler and faster reconstruction which should help enable non-Cartesian data sampling to be performed more widely in MRI.
### Provable Probabilistic Imaging using Score-Based Generative Priors
 - **Authors:** Yu Sun, Zihui Wu, Yifan Chen, Berthy T. Feng, Katherine L. Bouman
 - **Subjects:** Image and Video Processing (eess.IV); Computer Vision and Pattern Recognition (cs.CV); Machine Learning (cs.LG)
 - **Arxiv link:** https://arxiv.org/abs/2310.10835
 - **Pdf link:** https://arxiv.org/pdf/2310.10835
 - **Abstract**
 Estimating high-quality images while also quantifying their uncertainty are two desired features in an image reconstruction algorithm for solving ill-posed inverse problems. In this paper, we propose plug-and-play Monte Carlo (PMC) as a principled framework for characterizing the space of possible solutions to a general inverse problem. PMC is able to incorporate expressive score-based generative priors for high-quality image reconstruction while also performing uncertainty quantification via posterior sampling. In particular, we introduce two PMC algorithms which can be viewed as the sampling analogues of the traditional plug-and-play priors (PnP) and regularization by denoising (RED) algorithms. We also establish a theoretical analysis for characterizing the convergence of the PMC algorithms. Our analysis provides non-asymptotic stationarity guarantees for both algorithms, even in the presence of non-log-concave likelihoods and imperfect score networks. We demonstrate the performance of the PMC algorithms on multiple representative inverse problems with both linear and nonlinear forward models. Experimental results show that PMC significantly improves reconstruction quality and enables high-fidelity uncertainty quantification.
### Path Following Control of Automated Vehicle Considering Model  Uncertainties External Disturbances and Parametric Varying
 - **Authors:** Dan Shen
 - **Subjects:** Systems and Control (eess.SY)
 - **Arxiv link:** https://arxiv.org/abs/2310.10925
 - **Pdf link:** https://arxiv.org/pdf/2310.10925
 - **Abstract**
 Automated Vehicle Path Following Control (PFC) is an advanced control system that can regulate the vehicle into a collision-free region in the presence of other objects on the road. Common collision avoidance functions, such as forward collision warning and automatic emergency braking, have recently been developed and equipped on production vehicles. However, it is impossible to develop a perfectly precise vehicle model when the vehicle is driving. Most PFCs did not consider uncertainties in the vehicle model, external disturbances, and parameter variations at the same time. To address the issues associated with this important feature and function in autonomous driving, a new vehicle PFC is proposed using a robust model predictive control (MPC) design technique based on matrix inequality and the theoretical approach of the hybrid $\&$ switched system. The proposed methodology requires a combination of continuous and discrete states, e.g. regulating the continuous states of the AV (e.g., velocity and yaw angle) and discrete switching of the control strategy that affects the dynamic behaviors of the AV under different driving speeds. Firstly, considering bounded model uncertainties, and norm-bounded external disturbances, the system states and control matrices are modified.
### FusionU-Net: U-Net with Enhanced Skip Connection for Pathology Image  Segmentation
 - **Authors:** Zongyi Li, Hongbing Lyu, Jun Wang
 - **Subjects:** Image and Video Processing (eess.IV); Computer Vision and Pattern Recognition (cs.CV)
 - **Arxiv link:** https://arxiv.org/abs/2310.10951
 - **Pdf link:** https://arxiv.org/pdf/2310.10951
 - **Abstract**
 In recent years, U-Net and its variants have been widely used in pathology image segmentation tasks. One of the key designs of U-Net is the use of skip connections between the encoder and decoder, which helps to recover detailed information after upsampling. While most variations of U-Net adopt the original skip connection design, there is semantic gap between the encoder and decoder that can negatively impact model performance. Therefore, it is important to reduce this semantic gap before conducting skip connection. To address this issue, we propose a new segmentation network called FusionU-Net, which is based on U-Net structure and incorporates a fusion module to exchange information between different skip connections to reduce semantic gaps. Unlike the other fusion modules in existing networks, ours is based on a two-round fusion design that fully considers the local relevance between adjacent encoder layer outputs and the need for bi-directional information exchange across multiple layers. We conducted extensive experiments on multiple pathology image datasets to evaluate our model and found that FusionU-Net achieves better performance compared to other competing methods. We argue our fusion module is more effective than the designs of existing networks, and it could be easily embedded into other networks to further enhance the model performance.
### Analysis of potential flow networks: Variations in transport time with  $discrete$, $continuous$, and $selfish$ operation
 - **Authors:** Varghese Kurian, Sridharakumar Narasimhan
 - **Subjects:** Systems and Control (eess.SY)
 - **Arxiv link:** https://arxiv.org/abs/2310.10980
 - **Pdf link:** https://arxiv.org/pdf/2310.10980
 - **Abstract**
 In potential flow networks, the equilibrium flow rates are usually not proportional to the demands and flow control elements are required to regulate the flow. The control elements can broadly be classified into two types - discrete and continuous. Discrete control elements can have only two operational states: fully open or fully closed. On the other hand, continuous control elements may be operated in any intermediate position in addition to the fully open and fully closed states. Naturally, with their increased flexibility, continuous control elements can provide better network performance, but $to~what~extent$? We consider a class of branched networks with a single source and multiple sinks. The potential drop across edges ($\Delta H$) is assumed to be proportional to the $n^{th}$ power of flow rate ($Q$), i.e., $\Delta H=kQ^n$ , ($n>=1$). We define $\textbf{R}$ as the ratio of minimal operational times required to transport a given quantum of material with either type of control element and show that $1\leq \textbf{R}\leq m^{\left(1-1/n\right)}$, where $m$ is the maximum depth of the network. The results point to the role of network topology in the variations in operational time. Further analysis reveals that the selfish operation of a network with continuous control valves has the same bounds on the price of anarchy.
### $k$-$t$ CLAIR: Self-Consistency Guided Multi-Prior Learning for Dynamic  Parallel MR Image Reconstruction
 - **Authors:** Liping Zhang, Weitian Chen
 - **Subjects:** Image and Video Processing (eess.IV); Computer Vision and Pattern Recognition (cs.CV); Medical Physics (physics.med-ph)
 - **Arxiv link:** https://arxiv.org/abs/2310.11050
 - **Pdf link:** https://arxiv.org/pdf/2310.11050
 - **Abstract**
 Cardiac magnetic resonance imaging (CMR) has been widely used in clinical practice for the medical diagnosis of cardiac diseases. However, the long acquisition time hinders its development in real-time applications. Here, we propose a novel self-consistency guided multi-prior learning framework named $k$-$t$ CLAIR to exploit spatiotemporal correlations from highly undersampled data for accelerated dynamic parallel MRI reconstruction. The $k$-$t$ CLAIR progressively reconstructs faithful images by leveraging multiple complementary priors learned in the $x$-$t$, $x$-$f$, and $k$-$t$ domains in an iterative fashion, as dynamic MRI exhibits high spatiotemporal redundancy. Additionally, $k$-$t$ CLAIR incorporates calibration information for prior learning, resulting in a more consistent reconstruction. Experimental results on cardiac cine and T1W/T2W images demonstrate that $k$-$t$ CLAIR achieves high-quality dynamic MR reconstruction in terms of both quantitative and qualitative performance.
### Super resolution of histopathological frozen sections via deep learning  preserving tissue structure
 - **Authors:** Elad Yoshai, Gil Goldinger, Miki Haifler, Natan T. Shaked
 - **Subjects:** Image and Video Processing (eess.IV); Computer Vision and Pattern Recognition (cs.CV); Machine Learning (cs.LG)
 - **Arxiv link:** https://arxiv.org/abs/2310.11112
 - **Pdf link:** https://arxiv.org/pdf/2310.11112
 - **Abstract**
 Histopathology plays a pivotal role in medical diagnostics. In contrast to preparing permanent sections for histopathology, a time-consuming process, preparing frozen sections is significantly faster and can be performed during surgery, where the sample scanning time should be optimized. Super-resolution techniques allow imaging the sample in lower magnification and sparing scanning time. In this paper, we present a new approach to super resolution for histopathological frozen sections, with focus on achieving better distortion measures, rather than pursuing photorealistic images that may compromise critical diagnostic information. Our deep-learning architecture focuses on learning the error between interpolated images and real images, thereby it generates high-resolution images while preserving critical image details, reducing the risk of diagnostic misinterpretation. This is done by leveraging the loss functions in the frequency domain, assigning higher weights to the reconstruction of complex, high-frequency components. In comparison to existing methods, we obtained significant improvements in terms of Structural Similarity Index (SSIM) and Peak Signal-to-Noise Ratio (PSNR), as well as indicated details that lost in the low-resolution frozen-section images, affecting the pathologist's clinical decisions. Our approach has a great potential in providing more-rapid frozen-section imaging, with less scanning, while preserving the high resolution in the imaged sample.
### Zipformer: A faster and better encoder for automatic speech recognition
 - **Authors:** Zengwei Yao, Liyong Guo, Xiaoyu Yang, Wei Kang, Fangjun Kuang, Yifan Yang, Zengrui Jin, Long Lin, Daniel Povey
 - **Subjects:** Audio and Speech Processing (eess.AS); Machine Learning (cs.LG)
 - **Arxiv link:** https://arxiv.org/abs/2310.11230
 - **Pdf link:** https://arxiv.org/pdf/2310.11230
 - **Abstract**
 The Conformer has become the most popular encoder model for automatic speech recognition (ASR). It adds convolution modules to a transformer to learn both local and global dependencies. In this work we describe a faster, more memory-efficient, and better-performing transformer, called Zipformer. Modeling changes include: 1) a U-Net-like encoder structure where middle stacks operate at lower frame rates; 2) reorganized block structure with more modules, within which we re-use attention weights for efficiency; 3) a modified form of LayerNorm called BiasNorm allows us to retain some length information; 4) new activation functions SwooshR and SwooshL work better than Swish. We also propose a new optimizer, called ScaledAdam, which scales the update by each tensor's current scale to keep the relative change about the same, and also explictly learns the parameter scale. It achieves faster convergence and better performance than Adam. Extensive experiments on LibriSpeech, Aishell-1, and WenetSpeech datasets demonstrate the effectiveness of our proposed Zipformer over other state-of-the-art ASR models. Our code is publicly available at https://github.com/k2-fsa/icefall.
### Constrained FoV Radiated Power as a Figure of Merit of Phased Arrays
 - **Authors:** Alejandro Antón Ruiz, Samar Hosseinzadegan, John Kvarnstrand, Klas Arvidsson, Andrés Alayón Glazunov
 - **Subjects:** Systems and Control (eess.SY)
 - **Arxiv link:** https://arxiv.org/abs/2310.11361
 - **Pdf link:** https://arxiv.org/pdf/2310.11361
 - **Abstract**
 In this paper, we propose quantifying the radiated power of phased arrays or, in general, directive antennas, by the Constrained-View Radiated Power (CVRP). The constrained view shall be interpreted here as the Field-of-View (FoV) of an antenna that defines a region in space where focusing the radiated power is highly desired. In the limiting cases, we have that CVRP equals the Total Radiated Power (TRP) when the FoV covers the whole sphere, while, if the FoV reduces to a single point in space, the CVRP equals the Equivalent Isotropic Radiated Power (EIRP). We further present an analysis based on measured radiation patterns of a 16-element, linearly polarized, millimeter-Wave (mmWave), planar phased array antenna operating at 28 GHz. We compare the results to two ideal planar array antennas with the same number of Huygens and cosine elements. The evaluated figure of merit is computed for different scanning angles, as well as for different malfunctions of antenna elements, both for the real and simulated arrays. The results show that the introduced figure of merit could be potentially used for the detection of malfunctioning elements in antenna arrays as well as to characterize the impact of scan loss. Furthermore, CVRP is useful to straightforwardly and significantly characterize the performance of a directive antenna in terms of the power radiated towards a specific region in space.
